aci/resource_aci_dhcprelayp.go

```diff
@@ -81,16 +83,44 @@ func resourceAciDHCPRelayPolicy() *schema.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 						"addr": {
-							Type:         schema.TypeString,
-							Required:     true,
-							ValidateFunc: validation.IsIPv4Address,
+							Type:     schema.TypeString,
+							Required: true,
+							StateFunc: func(val interface{}) string {
+								splitVal := strings.Split(val.(string), "/")
```

Comment:
The input here shouldn't have a netmask so not sure we actually need this split and if/else.

Reply:
ok